Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.h

```diff
@@ -172,6 +174,7 @@ class VideoMediaSampleRenderer final : public ThreadSafeRefCountedAndCanMakeThre
     bool m_prefersDecompressionSession WTF_GUARDED_BY_CAPABILITY(mainThread) { false };
     std::optional<uint32_t> m_currentCodec;
     std::atomic<bool> m_gotDecodingError { false };
+    std::atomic<bool> m_needsFlushing { false };
```

Comment:
Is it only used/checked in main thread ?

Reply:
> Is it only used/checked in main thread ?

ah yes, I was setting in on the renderer queue in an earlier version.